src/Rel8/Type.hs

```diff
@@ -95,7 +97,12 @@ instance DBType Char where
   typeInformation = TypeInformation
     { encode = Opaleye.ConstExpr . Opaleye.StringLit . pure
     , decode = Hasql.char
-    , typeName = "char"
+    , typeName =
+        TypeName
+          { name = "bpchar"
+          , modifiers = ["1"]
+          , arrayDepth = 0
+          }
```

Comment:
Any reason to not just stay with `char`?

Reply:
Because the underlying type is actually called `bpchar`, not `char`. Now that we quote things properly, the SQL we produce for `lit '👍' will be `E'👍'::"char"`. However, in PostgreSQL, `char` and `"char"` are different types, with the latter meaning [a single ASCII character](https://www.postgresql.org/docs/current/datatype-character.html), so if we use don't change to `bpchar` then `lit '👍' == lit '👎'`.